tox.ini

```diff
@@ -34,16 +35,16 @@ deps =
     django4.1: Django~=4.1
     djangomain: git+https://github.com/django/django.git@main#egg=Django
 
-    wagtail2.15: wagtail>=2.15,<2.16
-    wagtail2.16: wagtail>=2.16,<2.17
-    wagtail3.0: wagtail>=3.0,<4.0
-    wagtail4.0: wagtail>=4.0,<4.1
-    wagtail4.1: wagtail>=4.1,<5.0
+    wagtail4.1: wagtail>=4.1,<4.2
+    wagtail4.2: wagtail>=4.2rc1,<5.0
     wagtailmain: git+https://github.com/wagtail/wagtail.git
 
     # Pinned to work around UTC connection error
     # https://stackoverflow.com/questions/68024060/assertionerror-database-connection-isnt-set-to-utc
-    postgres: psycopg2==2.8.6
+    postgres12: psycopg2==2.8.6
+
+    # https://www.psycopg.org/docs/index.html
+    postgres15: psycopg2==2.9.5
```

Comment:
- Created another instance of postgres:

1. postgres12 - uses `pyscopg2==2.8.6` which supports PostgreSQL server versions from 7.4 to 12 (https://access.crunchydata.com/documentation/psycopg2/2.8.6/install.html)
2. postgres15 - uses `pyscopg2==2.9.5` which supports PostgreSQL server versions from 7.4 to 15 (https://www.psycopg.org/docs/install.html#prerequisites)

As suggested from this conversation: https://torchbox.slack.com/archives/C03R0FNK7/p1675655322697539

Reply:
HI @katdom13 It didn't look like the test were really running with a postgres DB.

I have investigated it and made a PR to your branch: https://github.com/katdom13/wagtail-localize/pull/3 which should actually set up the postgres container and pass the required DB env vars.